examples/src/test/scala/akka/projection/cassandra/scaladsl/WordCountDocExampleSpec.scala

```diff
@@ -123,6 +124,30 @@ class WordCountDocExampleSpec
       runAndAssert(projection)
     }
 
+    "have support for actor Behavior as handler" in {
+      import IllstrateActorLoadingStateOnDemand._
+
+      val projectionId = genRandomProjectionId()
+
+      //#actorHandler
+      val handler = ActorHandler(
+        projectionId,
+        WordCountProcessor(projectionId, repository),
+        envelopeMessage = (env, replyTo) => WordCountProcessor.Handle(env, replyTo))
```

Comment:
I started by only documenting how to do this, without the `ActorHandler`, but I think it's nice to include the ActorHandler. Otherwise it would look like:
```
    class WordCountHandler(projectionId: ProjectionId, repository: WordCountRepository)(implicit system: ActorSystem[_])
        extends Handler[WordEnvelope] {
      import akka.actor.typed.scaladsl.AskPattern._
      import system.executionContext
      private implicit val timeout: Timeout = 10.seconds

      private val processor: Future[ActorRef[WordCountProcessor.Command]] =
        SpawnExtension(system).spawn(WordCountProcessor(projectionId, repository), projectionId.id)
      private val terminated = processor.flatMap(ref => WatchExtension(system).watch(ref))

      override def process(envelope: WordEnvelope): Future[Done] = {
        processor.flatMap(_.ask[Try[Done]](replyTo => WordCountProcessor.Handle(envelope, replyTo))).map {
          case Success(_)   => Done
          case Failure(exc) => throw exc
        }
      }

      override def stop(): Future[Done] = {
        processor.foreach(_ ! WordCountProcessor.Stop)
        terminated
      }
    }
```